src/css/flags/audioplayer.less

```diff
@@ -43,3 +43,9 @@
 .jwplayer.jw-flag-audio-player .jw-preview {
     display: none;
 }
+
+.jwplayer.jw-flag-media-audio.jw-flag-audio-player {
```

Comment:
We only care if the player is on audio-mode (ie , the audio-player flag), thus the media-audio flag should not be here.